src/features/broker-protection.js

```diff
@@ -0,0 +1,21 @@
+import ContentFeature from '../content-feature.js'
+import { execute } from './broker-protection/execute.js'
+
+export default class BrokerProtection extends ContentFeature {
+    init () {
+        this.messaging.subscribe('onActionReceived', (/** @type {any} */params) => {
+            try {
+                const action = params.state.action
+                const data = params.state.data
+                if (!action) {
+                    return this.messaging.notify('actionError', { error: 'No action found.' })
+                }
+                const result = execute(action, data)
+                this.messaging.notify('actionCompleted', { result })
```

Comment:
does the return value matter in the `subscribe()` callback? The line 11 returns something, but this does not

Reply:
No, just an early exit.